frontend/src/app/store/effects/dashboard/tasks.effects.ts

```diff
@@ -41,5 +42,37 @@ export class TasksEffects {
         }
     );
 
+    public closeDisplaySidebar$ = createEffect(
```

Comment:
Why is it in tasks effects? Why taskCreated and taskEdited in dashboardActions? Why those are not tasks actions?
Where do you store tasks? In Dashboard? Shouldn't it be like: 2 separate stores so tasks no nothing about dashboard and dashboard may know about tasks because it contains tasks? And it would mean that dashboard react on tasks actions and close sidebar. So task would only know how to be created and updated.

it is valid for every store, reducer, action and effect - review them. Decide who and what really should be responsible for. if there is a need we may discuss

Reply:
Separated task and dashboard states